latch_cli/centromere/utils.py

```diff
@@ -165,24 +168,41 @@ def _from_env():
             ) from de
 
 
-def _construct_ssh_client(host_ip: str, username: str):
+def _construct_ssh_client(internal_ip: str, username: str) -> paramiko.SSHClient:
+    gateway = paramiko.SSHClient()
+    gateway.load_system_host_keys()
+    gateway.set_missing_host_key_policy(paramiko.MissingHostKeyPolicy)
+    gateway.connect(latch_constants.jump_host, username=latch_constants.jump_user)
+
+    transport = gateway.get_transport()
+    if transport is None:
+        raise ConnectionError("unable to create connection to jump host")
+    sock = transport.open_channel(
+        kind="direct-tcpip",
+        dest_addr=(internal_ip, 22),
+        src_addr=("", 0),
+    )
+
     ssh = paramiko.SSHClient()
     ssh.load_system_host_keys()
-    ssh.set_missing_host_key_policy(paramiko.MissingHostKeyPolicy())
-    ssh.connect(host_ip, username=username)
+    ssh.set_missing_host_key_policy(paramiko.MissingHostKeyPolicy)
+    ssh.connect(internal_ip, username=username, sock=sock)
     return ssh
 
 
 class _TmpDir:
 
     """Represents a temporary directory that can be local or on a remote machine."""
 
-    def __init__(self, ssh_client=None, remote=False):
+    def __init__(self, ssh_client: Optional[paramiko.SSHClient] = None, remote=False):
         if remote and not ssh_client:
```

Comment:
```suggestion
        if remote and ssh_client is None:
```
no truthy comparisons